src/lessons/08_grid_layout/grid-layout.slides.html

```diff
@@ -0,0 +1,758 @@
+                    <li class="fragment"><code>justify-items</code>: Align grid items along row axis</li>
+                    <li class="fragment"><code>justify-self</code>: Align single grid item along row axis</li>
+                    <li class="fragment"><code>align-items</code>: Align grid items along column axis</li>
+                    <li class="fragment"><code>align-self</code>: Align single grid item along column axis</li>
+                </ul>
+                <p class="fragment">Values: start | end | center | stretch</p>
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+      <code class="formattedCode">
+          # CSS
+            .grid-container {
+              grid-template-columns: auto auto auto;
+              grid-template-rows: 80px 90px;
+              justify-items: end
+            }
+            .grid-item-1 {
+              justify-self: start;
+              align-self: center;
+            }
+
+          # HTML
+          <div class="grid-container">
+            <div class="grid-item-1">Item 1</div>
+            <div class="grid-item-2">Item 2</div>
+            <div class="grid-item-3">Item 3</div>
+            <div class="grid-item-4">Item 4</div>
+            <div class="grid-item-5">Item 5</div>
+          </div>
+  </code><br><div class="fragment grid-container grid-13">
+    <div class="grid-item-1">Item 1</div>
+    <div class="grid-item-2">Item 2</div>
+    <div class="grid-item-3">Item 3</div>
+    <div class="grid-item-4">Item 4</div>
+    <div class="grid-item-5">Item 5</div>
+</div></pre>
+            </section>
+
+            <section>
+                <h3>Content Placement in Container</h3>
+                <br>
+                <p><b>Properties</b></p>
+                <ul>
+                    <li class="fragment"><code>justify-content</code>: Align tracks along row axis</li>
+                    <li class="fragment"><code>align-content</code>: Align tracks along column axis</li>
+                </ul>
+
+                <p class="fragment">Values: start | end | center | stretch | space-around | space-between | space-evenly</p>
+
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+                  <code class="formattedCode">
+                    # CSS
+                      .grid-container {
+                        grid-template-columns: auto auto auto;
+                        grid-template-rows: 80px 90px;
+                        justify-content: end
+                      }
+          
+                    # HTML
+                    <div class="grid-container">
+                      <div class="grid-item">Item 1</div>
+                      <div class="grid-item">Item 2</div>
+                      <div class="grid-item">Item 3</div>
+                      <div class="grid-item">Item 4</div>
+                      <div class="grid-item">Item 5</div>
+                    </div>
```

Comment:
Maybe is better to separate this into 2 block codes so the formatting is correct

Reply:
[Fixed](https://github.com/ReDI-Munich-Frontend/fed-munich-2021-spring/commit/1c4091a9316975c61acb733fbfdb941f98d3266a)